extensions/amp-dailymotion/0.1/validator-amp-dailymotion.protoascii

```diff
@@ -64,6 +64,7 @@ tags: {  # <amp-dailymotion>
     mandatory: true
     value_regex_casei: "[a-z0-9]+"
   }
+  attrs: { name: "autoplay" }
```

Comment:
We attempt to keep the attrs in alphabetical order, so please move this up above `data-endscreen-enable`